snapshots/dummy/2020-01-01/dummy_full.csv.dvc

```diff
@@ -1,16 +1,16 @@
 meta:
   origin:
-    dataset_title_owid: '[origin.dataset-title-owid]'
-    dataset_title_producer: '[origin.dataset-title-producer]'
-    dataset_description_owid: '[origin.dataset-description-owid]'
-    dataset_description_producer: '[origin.dataset-description-producer]'
+    title: '[origin.titla]'
```

Comment:
Typo. Also, in this file, again, I changed `_` to `-` to avoid italics in data pages.